### DISCUSSION TITLE
Can I implement this fully offline with local LLM

### DISCUSSION THREAD

**Discussion by pradhandebasish2046** [microsoft/graphrag#335] [General]
Hii,
I want to use graphRAG in my private dataset fully offline. Is there any way I can do it without calling open ai API key

**Comment by AlonsoGuevara** [comment]
Hi! We are centralizing other LLM discussions in these threads:
Other LLM/Api bases: #339,
Ollama: #345
Local embeddings: #370 

I'll resolve this issue so we can keep the focus on those threads
